src/style.css

```diff
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.search {
+  margin-top: 20px;
+  overflow: hidden;
+  position: relative;
+}
+
+.input::placeholder {
+  font-weight: 500;
+  color: #3d4e61;
+}
+  
+.input {
+  box-sizing: border-box;
+  position: relative;
+  outline: none;
+  border: 1px solid #e1e7ed;
+  width: 100%;
+  font-family: "Avenir", Arial, sans-serif;
```

Comment:
It would be better to move `font-family` to the `body` selector,
and use `font-family: inherit` everywhere you need